source/ide/ide_methods.bas

```diff
@@ -313,7 +313,7 @@ FUNCTION ide2 (ignore)
         menu$(m, i) = "-": i = i + 1
 
         RunMenuSaveExeWithSource = i
-        menu$(m, i) = "Output EXE to Source #Folder": i = i + 1
+        menu$(m, i) = "Output executable to source #folder": i = i + 1

```

Comment:
This is just a matter of personal preference with this change in particular

Reply:
Technically exe *is* wrong

Reply:
If they are that technical then they shouldn't think we are creating an actual EXE file. EXE means executable. Abbreviations are found in many places in programming and general IT.